File: TransCoder_tokenized_test_set_functions/java/DIFFERENCE_MAXIMUM_SUM_MINIMUM_SUM_N_M_ELEMENTSIN_REVIEW.java
static int find_difference ( int arr [ ], int n, int m ) {
  int max = 0, min = 0;
  Arrays . sort ( arr );
  for ( int i = 0, j = n - 1;
  i < m;
  i ++, j -- ) {
    min += arr [ i ];
    max += arr [ j ];
  }
  return ( max - min );
}

File: results/py/DIFFERENCE_MAXIMUM_SUM_MINIMUM_SUM_N_M_ELEMENTSIN_REVIEW.cpp
int f_filled ( int * arr , int n , int m ) {
  int max = 0 ;
  ;
  int min = 0 ;
  sort ( arr , arr + m ) ;
  int j = n - 1 ;
  for ( int i = 0 ;
  i < m ;
  i ++ ) {
    min += arr [ i ] ;
    max += arr [ j ] ;
    j = j - 1 ;
  }
  return ( max - min ) ;
}
